### ISSUE TITLE
Add Front Page

### ISSUE THREAD

**Issue by varkud** [MaxMelcher/AzureDevOps.WikiPDFExport#64]
Would be good to have an option to select a front page, either as:
- html(the same as footer/header)
- Parent page (the same as folder name on one level up)
- path to wiki page

p.s. superb functionality!! Thanks for your work.

**Comment by MaxMelcher** [comment]
So a structure like this? 
[FrontPage]
[the Wiki]

What is the scenario for this? 
Why is the frontpage not in the wiki?

**Comment by Jheronymus** [comment]
I was thinking of the same thing. But I think a nice front page, possibly with full size logo, big title etc would be nice as a document cover. While having such a thing as an opening for a wiki might be a bit weird.
some wiki's/ docs-as-code type of solutions allow for some meta data on the page. Maybe that is a good way to allow for flexibility between wiki and document?

**Comment by Diom** [comment]
@MaxMelcher The --glocaltoc option give this new importance, as we want the title page before the TOC.

**Comment by Diom** [comment]
By the way on this... with the last commit the TOC was wrapped in a `<div class='toc'>...</div>` container. I was able to add a title page by using CSS:

~~~css
.toc {
    page-break-after: always;
}

h1.title {
    page-break-before: avoid !important;
    page-break-after: always;
    padding-top: 25%;
    text-align: center;
    font-size: 48px;
}
~~~

and the `--globaltoc` option set with HTML:
`--globaltoc "<h1 class='title'>My Title</h1>"`

This works well for me anyways. Means I can set the title page at generation time.

**Comment by cpetry** [comment]
I am now also in need of this.
So here are two options:
- Define a front page via markdown and name it as a --frontpage parameter
- Define the front page via html like @Diom directly in parameters

I guess as it takes a little while to polish this front page a markdown approach would be nicest (at least in my case).
I'll create a PR for this

**Comment by MaxMelcher** [comment]
I agree. Markdown is the way to go, adding a custom html-based frontpage would be too much.

thanks @cpetry

**Comment by nrogoff** [comment]
Would be good to include the 'folder' level markdown page. e.g. if exporting the path and subfolders below ```/MyProject/Architecture``` then the front page would be ```/MyProject/Architecture.md```
